packages/semver/src/schematics/ng-add/index.ts

```diff
@@ -37,6 +44,11 @@ export function ngAdd(options: SchemaOptions): Rule {
               ? updateWorkspaceFromSchema(options)
               : await updateWorkspaceFromPrompt(tree),
           ]),
+      options.enforceConventionalCommits ? addDependencies(options) : noop(),
+      options.enforceConventionalCommits ? addCommitizenConfig() : noop(),
+      options.enforceConventionalCommits ? addCommitlintConfig() : noop(),
+      options.enforceConventionalCommits ? addHuskyConfig() : noop(),
+      options.enforceConventionalCommits ? addHuskyConfigMsg() : noop(),
```

Comment:
I prefer the old way with only one if statement.

Reply:
done